pymc_marketing/mmm/base.py

```diff
@@ -996,147 +979,34 @@ def _plot_response_curve_fit(
 
         # Generate x_fit and y_fit
         x_fit = np.linspace(0, x_limit, 1000)
-        y_fit = fit_function(x_fit, alpha_limit, lam_constant)
-        y_fit_lower = fit_function(x_fit, alpha_limit_lower, lam_constant_lower)
-        y_fit_upper = fit_function(x_fit, alpha_limit_upper, lam_constant_upper)
+        upper_params = self.format_recovered_transformation_parameters(quantile=.95)
+        lower_params = self.format_recovered_transformation_parameters(quantile=.05)
```

Comment:
can we pass these quantiles as function arguments? Say something like `quantile_lower` or `quantile_lwr` and `quantile_upr`?